Lib/pydoc.py

```diff
@@ -288,6 +291,9 @@ def visiblename(name, all=None, obj=None):
     # Namedtuples have public fields and methods with a single leading underscore
     if name.startswith('_') and hasattr(obj, '_fields'):
         return True
+    if name in _future_feature_names:
```

Comment:
Perhaps a comment would help. This looks confusing on its own.
```suggestion
    # Ignore __future__ imports.
    if name in _future_feature_names:
```